src/Views/Form/Templates/Sequoia/assets/js/form.js

```diff
@@ -196,6 +196,26 @@
 					$( '.give-fee-recovery-donors-choice' ).toggleClass( 'active' );
 				} );
 
+				// If mailchimp is opted in by default, add border on load
+				if ( $( 'input[name="give_mailchimp_signup"]' ).prop( 'checked' ) === true ) {
+					$( '.give-mailchimp-fieldset' ).addClass( 'active' );
+				}
+
+				// Persist mailchimp input border when selected
+				$( '.give-mc-message-text' ).on( 'click touchend', function() {
+					$( '.give-mailchimp-fieldset' ).toggleClass( 'active' );
+				} );
+
+				// If mailchimp is opted in by default, add border on load
```

Comment:
Typo: your comment says mailchimp but this has to do with CC.

Reply:
Good catch, fixed: https://github.com/impress-org/givewp/commit/bddd8865e2ee87773bb1bb9884a0a47fa5be120c